src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -668,18 +675,30 @@ fun selectEntitySetWithPropertyTypesAndVersion(
     //@formatter:off
     val columns = setOf(
             ID_VALUE.name,
-            if(metadataOptions.contains(MetadataOption.LAST_WRITE) ) { LAST_WRITE.name } else { "" },
```

Comment:
columns = setOf(ID_VALUE.name) + metadataOptions.map { it.name }  + authorizedPropertyTypeValues.map(::quote)

Reply:
Shouldn't we use the Postgres...Definitions for names instead of metadataoptions?

Reply:
Yes. The idea was MetadataOption class is in API while Postgres...Definitions only exist server side. You'll have to transform what the client passes (MetadataOption) into Postgres...Definition classes to do this.